### PR TITLE
HPCC-13259 Add super owner information

### PR DESCRIPTION
**Pull request by miguelvazq**
Give the user super owner information in the logical files list and logical files details pages.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review. @wangkx Please add super owner to the file view response. There is no way to pass in "IncludeSuperOwners". JIRA https://track.hpccsystems.com/browse/HPCC-16925

**Comment by miguelvazq** [comment]
Modifying this in case user uses a shortcut to get to DFUDetailsWidget

File: esp/src/eclwatch/LFDetailsWidget.js [hunk 1]
```diff
@@ -367,6 +367,8 @@ define([
                 this.updateInput("DFUFilePartsOnClusters", oldValue, newValue.DFUFilePartsOnCluster[0].Replicate);
             } else if (name === "RecordSize" && newValue === "0") {
                 this.updateInput("RecordSize", oldValue, this.i18n.NoPublishedSize);
+            } else if (name === "SuperOwners") {
+                this.updateInput("SuperOwner", oldValue, this.params.SuperOwner);
```
Comment (GordonSmith) [inline esp/src/eclwatch/LFDetailsWidget.js]: Shouldn't this be ```newValue``` instead of ```this.params.SuperOwner```?

**Review by GordonSmith — COMMENTED** [review]
One question.

**Comment by miguelvazq** [comment]
@GordonSmith updated. I changed from passing in parameters to pulling out the items from super files in LFDetails in case they come in from a shortcut. There can be multiple super owners so an array is perfect for this.

**Review by GordonSmith — APPROVED** [review]
Looks ok.

**Pull request merged**